ui/packages/catalog-ui-search/src/main/webapp/component/input/radio/input-radio.view.js

```diff
@@ -56,12 +56,14 @@ define([
                         if (value.label) {
                             return {
                                 label: value.label,
-                                value: value.value
```

Comment:
❓ do we need the title?

Reply:
I'm guessing this is for possible use by downstream extensions.

Reply:
We are using the value in the [radio](https://github.com/codice/ddf/pull/3230/files#diff-e7bb115f147c2f342e1caffb89d98685R15) component.